conf/master

```diff
@@ -318,17 +326,18 @@
 # and retrieved via sdb. See https://docs.saltstack.com/en/latest/topics/sdb/.
 # Passphrase protection is off by default but an example of an sdb profile and
 # query is as follows.
-# masterkeyring:
+#masterkeyring:
 #  driver: keyring
 #  service: system
 #
-# key_pass: sdb://masterkeyring/key_pass
-
+#key_pass: sdb://masterkeyring/key_pass
+#
 # Enable passphrase protection of the Master signing_key. This only applies if
 # master_sign_pubkey is set to True.  This is disabled by default.
-# master_sign_pubkey: True
-# signing_key_pass: sdb://masterkeyring/signing_pass
+#master_sign_pubkey: False
 
```

Comment:
why are we adding this newline between

Reply:
To remove ambiguity - this option is the default, hence a blank line after it.